platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/config/ReconnectConfig.java

```diff
@@ -54,7 +54,7 @@ public record ReconnectConfig(
         @ConfigProperty(defaultValue = "true") boolean active,
         @ConfigProperty(defaultValue = "-1") int reconnectWindowSeconds,
         @ConfigProperty(defaultValue = "0.50") double fallenBehindThreshold,
-        @ConfigProperty(defaultValue = "60s") Duration asyncStreamTimeout,
+        @ConfigProperty(defaultValue = "180s") Duration asyncStreamTimeout,
```

Comment:
This is not enough. The value is explicitly set per environment, e.g. in `hedera-node/configuration/mainnet/settings.txt` They should be changed, too (or better removed, since we usually don't have overrides, which are identical to default values)

Reply:
in the `configuration/*/settings.txt` files, the old setting name is specified (`reconnect.asyncStreamTimeoutMilliseconds, 60000`), so i dont think that value is even being used. It should be removed from all the `settings.txt` files.

Reply:
fair enough. I'll fix it

Reply:
Done